### PR TITLE
Include host inside event. We need port and IP

### PR DESCRIPTION
**Pull request by podviaznikov**
We need `host` parameter. `ip` is not enough.

### CODE REVIEW

File: lib/events.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+require('loadenv.js')();
+var ip = require('ip');
+var uuid = require('node-uuid');
+
+// add `ip`, `uuid`, `host` and `time` (if missing) fields
+exports.enhanceEvent = function (ev) {
+  ev.ip = ip.address();
+  ev.uuid = uuid.v1();
+  if (ev.time) {
```
Comment (bkendall) [inline lib/events.js]: this condition seems backwards...

Comment (podviaznikov) [inline lib/events.js]: Fixed and added tests.

**Comment by bkendall** [comment]
lgtm!

**Comment by bkendall** [comment]
go ahead and merge away!

**Comment by tjmehta** [comment]
:+1:

**Pull request merged**